src/defaultsettings.cpp

```diff
@@ -464,7 +465,6 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("screen_h", "0");
 	settings->setDefault("fullscreen", "true");
 	settings->setDefault("touchtarget", "true");
-	settings->setDefault("TMPFolder", porting::getDataPath("tmp" DIR_DELIM));
```

Comment:
This setting is undocumented, and can also be removed from common.lua. You can remove the whole if statement:

```lua
os.tempfolder = function()
	if core.settings:get("TMPFolder") then
		return core.settings:get("TMPFolder") .. DIR_DELIM .. "MT_" .. math.random(0,10000)
	end
```

Reply:
pkgmgr.lua:177